### PR TITLE
Added a comment: "Finish what you start, bro"

### PR DESCRIPTION
**Pull request by jpark712**

### CODE REVIEW

File: Sorting.java [hunk 1]
```diff
@@ -1,3 +1,7 @@
+//
+// Finish what you start bro
```
Comment (darbinreyes) [inline Sorting.java]: Forgot a period.

**Review by darbinreyes — CHANGES_REQUESTED** [review]
Test 2.

File: Sorting.java (hunk 1 above)
Comment (jpark712) [reply Sorting.java]: Oops my bad

**Review by darbinreyes — APPROVED** [review]
Good job, mang.

**Pull request merged**